pkg/lease/v1alpha1/csp_lease.go

```diff
@@ -19,16 +19,17 @@ package lease
 import (
 	"encoding/json"
 	"fmt"
+	"strings"
+
 	"github.com/golang/glog"
 	apis "github.com/openebs/maya/pkg/apis/openebs.io/v1alpha1"
 	env "github.com/openebs/maya/pkg/env/v1alpha1"
-	"github.com/openebs/maya/pkg/patch/v1alpha1"
-	"k8s.io/api/core/v1"
+	patch "github.com/openebs/maya/pkg/patch/v1alpha1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
IMO this alias is not required since it is same as the last folder name

Reply:
Better to use `corev1` as alias

Reply:
done